CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ We welcome direct contributions to the rest code base. Thank you!
 
 ##### Supported Versions #####
 
-- Go version 1.14, 1.15 or 1.16
+- Go version 1.14, 1.15, 1.16, 1.17, 1.18, or 1.19
```

Comment:
Let's include `1.20`, `1.21` will be released this month too.

Reply:
Ok, Go 1.20 has been added.